regenerate.py

```diff
@@ -56,29 +54,44 @@ def output(input_path, output_path):
     output(doc_path, output_path)
 
 
+def CopyCss(css_dir: str, site_root: str):
+  output_dir = os.path.join(site_root, css_dir)
+  os.makedirs(output_dir, exist_ok=True)
+  input_files = glob.iglob(os.path.join(css_dir, '*.css'))
+  for input_file in input_files:
+    shutil.copy2(input_file, output_dir)
+
+
 def ParseArgs():
   parser = argparse.ArgumentParser(
     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-  parser.add_argument('--posts',
-                      type=str,
-                      default='algos',
-                      help='directory relative to the repo root')
-  parser.add_argument('--docs',
-                      type=str,
-                      default='docs',
-                      help='directory relative to the repo root')
-  parser.add_argument('--site_root',
-                      type=str,
-                      default='build',
-                      help='output dir, relative to the repo root')
-  parser.add_argument('--pandoc_flags',
-                      type=str,
-                      default='',
-                      help='passed verbatim to pandoc')
-  parser.add_argument('--log_level',
-                      type=str,
-                      help='controls the script\'s logging level',
-                      default='ERROR')
+  parser.add_argument(
```

Comment:
My formatting for these changes is also like the old version.

Reply:
Fixed.